relay-system/Cargo.toml

```diff
@@ -17,7 +17,7 @@ futures = { workspace = true }
 once_cell = { workspace = true }
 relay-log = { workspace = true }
 relay-statsd = { workspace = true }
-tokio = { workspace = true, features = ["rt", "signal"] }
+tokio = { workspace = true, features = ["rt", "signal", "macros"] }
```

Comment:
What is this for?

Reply:
It fixes a compilation issue when the relay-system crate is compiled alone (e.g. to run tests). The crate actually depends on macros, we just didn't notice before as another workspace member enables this feature usually.